package.json

```diff
@@ -1,23 +1,32 @@
 {
-  "name": "math-magicians",
+  "name": "bull-cow-game",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.2.0",
     "@testing-library/user-event": "^13.5.0",
+    "big.js": "^6.1.1",
     "prop-types": "^15.8.1",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
-    "react-scripts": "^5.0.1",
+    "react-scripts": "5.0.1",
+    "uniqid": "^5.4.0",
+    "util": "^0.12.4",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "DISABLE_ESLINT_PLUGIN=true  react-scripts start",
```

Comment:
# Optional Changes 
All changes marked as optional cannot stop your project from being approved but will help improve your code and make you become a better programmer writing clean and professional code. 👍 

- [ ]  Kindly remove this command as it's stopping the app script from starting. I had to remove it before the app can start.

**See the feedback I get from the terminal**
![image](https://user-images.githubusercontent.com/50941074/170233417-050a2133-09fd-42d3-bb0d-6d4b8110e36f.png)